internal/build/helper.go

```diff
@@ -7,20 +7,20 @@ import (
 	ootov1alpha1 "github.com/qbarrand/oot-operator/api/v1alpha1"
 )
 
-//go:generate mockgen -source=modulehelper.go -package=build -destination=mock_modulehelper.go
+//go:generate mockgen -source=helper.go -package=build -destination=mock_helper.go
 
-type ModuleHelper interface {
+type Helper interface {
 	ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides ...v1alpha1.BuildArg) []v1alpha1.BuildArg
 	GetReleventBuild(mod ootov1alpha1.Module, km ootov1alpha1.KernelMapping) *ootov1alpha1.Build
 }
 
-type moduleHelper struct{}
+type helper struct{}
 
-func NewModuleHelper() ModuleHelper {
-	return moduleHelper{}
+func NewHelper() Helper {
+	return helper{}
 }
 
-func (m moduleHelper) ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides ...v1alpha1.BuildArg) []v1alpha1.BuildArg {
+func (m helper) ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides ...v1alpha1.BuildArg) []v1alpha1.BuildArg {
```

Comment:
```suggestion
func (m *helper) ApplyBuildArgOverrides(args []v1alpha1.BuildArg, overrides ...v1alpha1.BuildArg) []v1alpha1.BuildArg {
```

And same for all other methods of `helper`.

Reply:
fixed